source/blog/2016-05-25-ember-data-2-6-released.md

```diff
@@ -0,0 +1,129 @@
+
+Enables a new `ids-and-type` strategy (in addition to the already existing `ids` and `records`) for
+serializing has many relationships using the `DS.EmbeddedRecordsMixin` that  will include both
+`id` and `type` of each model as an object.
+
+For instance, if a user has many pets, which is a polymorphic relationship, the generated payload would be:
+
+```js
+{
+  "user": {
+    "id": "1"
+    "name": "Bertin Osborne",
+    "pets": [
+      { "id": "1", "type": "Cat" },
+      { "id": "2", "type": "Parrot"}
+    ]
+  }
+}
+```
+
+This is particularly useful for polymorphic relationships not backed
+  by [STI](https://en.wikipedia.org/wiki/Single_Table_Inheritance)
+  when just including the id of the records is not enough.
+
+Thanks to [@cibernox](https://github.com/cibernox) for
+implementing this feature.
+
+### Upcoming Features in Ember Data 2.7.beta-1
+
+#### `ds-boolean-transform-allow-null` [#4022](https://github.com/emberjs/data/pull/4022)
+
+Allow null/undefined values for boolean attributes via `DS.attr('boolean', { allowNull: true })`
+
+```js
+// app/models/user.js
+
+export default DS.Model.extend({
+  email: DS.attr('string'),
+  username: DS.attr('string'),
+  wantsWeeklyEmail: DS.attr('boolean', { allowNull: true })
+});
+```
+
+#### `ds-improved-ajax` [#3099](https://github.com/emberjs/data/pull/3099)
+
+Though `ajax()` (and `ajaxOptions()`) of `DS.RESTAdapter` are marked as
+private, they have been overwritten in many applications, since there is
+currently no other way to customize the request.
+
+This feature adds new public methods to `DS.RESTAdapter`, which allow to
+customize the properties of a request:
+
+- `methodForRequest` to get the HTTP verb
+- `urlForRequest` to get the URL
+- `headersForRequest` to get the headers
+- `dataForRequest` to get the data (query params or request body)
+
+The `params` hash passed to those methods has all the properties with
+which the corresponding `find`, `createRecord`, `findQuery`, ...  call
+have been invoked: store, type, snapshot(s), id(s) and query. The
+`requestType` property indicates which method is requested; the possible
+values are:
+
+- `createRecord`
+- `updateRecord`
+- `deleteRecord`
+- `query`
+- `queryRecord`
+- `findRecord`
+- `findAll`
+- `findMany`
+- `findHasMany`
+- `findBelongsTo`
+
+Performing the actual AJAX request is handled by the `makeRequest`
+method, which is similar to the existing `ajax` method: it makes the
+request using `jQuery.ajax` and attaches success and failure handlers.
+
+---
+
+Say your API handles creation of resources via PUT, this can now be
+customized as follows:
+
+``` js
+// adapters/application.js
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  methodForRequest: function(params) {
+    if (params.requestType === 'createRecord') {
+      return "PUT";
+    }
+
+    return this._super(...arguments);
+  }
+});
+```
+
+Thanks to [@pangratz](https://github.com/pangratz) for
+implementing this feature.
```

Comment:
It'd be great to get all this info in the guides on the adapters page as well ...  Seems like we could almost lift this verbatim to make it easier for folks to find instead of looking on the blog?